smoke-tests/smoke-test.cmd

```diff
@@ -31,4 +32,4 @@ call %EXE_NAME% scanner:run --format junit --category SomeCat1,Security --target
 echo "==== Remove a custom rule ===="
 call %EXE_NAME% scanner:rule:remove --path test\test-jars\apex\testjar1.jar --force || exit /b 1
 echo "==== List the rules a final time, to make sure nothing broke ===="
-call %EXE_NAME% scanner:rule:list || exit /b 1
\ No newline at end of file
+call %EXE_NAME% scanner:rule:list || exit /b 1
```

Comment:
Just curious, any idea why the newline changed? There is nothing obvious in the script that I could see.

Reply:
I did the edits to the smoke test scripts manually this time, because regenerating the files can mess with the permissions and I didn't want to deal with that for such a simple change. I think the IDE manually added the newline at the end. It does that sometimes.